src/widgets/voice-search/voice-search.tsx

```diff
@@ -113,16 +110,26 @@ const voiceSearch: VoiceSearch = widgetParams => {
     button: cx(suit({ descendantName: 'button' }), userCssClasses.button),
     status: cx(suit({ descendantName: 'status' }), userCssClasses.status),
   };
+  const templates = {
+    ...defaultTemplates,
+    ...userTemplates,
+  };
```

Comment:
isn't this code already done by prepareTemplateProps? maybe we should pass templates, defaultTemplates to the renderer, and let prepareTemplateProps deal with it. Does that give the same outcome?

Reply:
This indeed gives the same outcome, but some widgets don't yet use `prepareTemplateProps`, or/and don't have typed `renderer` params. I think it would be best to handle all of this in a PR where we type the `renderer` params and use `prepareTemplateProps` at the same time. 

wdyt? 

wdyt?

Reply:
and still here

Reply:
This one doesn't have the `prepareTemplateProps` so I've kept the spread here